spec/javascript/testUtils/wrapperUtil.js

```diff
@@ -0,0 +1,78 @@
+import React from 'react'
+import { shallow, mount } from 'enzyme'
+import Context from '~/components/supplemental_application/context'
+import { Form } from 'react-final-form'
+
+const formNode = (application, formToChildrenFunc) => (
```

Comment:
formToChildrenFunc isn't super intuitive for me. What about something like componentWithForm?

Reply:
@akegan how about renderWithForm? That way it kind of matches with react-final-form's `render` prop

Reply:
oh I see. I had missed before that we have to pass in (form) => <Component form={form} /> instead of just the component. Maybe formToChildrenFunc does actually make the most sense then.

Reply:
would it be possible to just pass the component and then in the formNode function pass in the form to the component?

Reply:
@akegan it would (I think, never added a prop to a component after it's been created) but i'd rather we didn't do that.

It would totally work for what we're doing here, but if a component ever has a prop with a different name, or a component just uses a single field of the form this wont work.

Another thing I'm thinking we'll need eventually is to add support for the other final-form callback params (I think `values` is one?), and it would be easiest to just add it to the render func instead of adding them all as props to the child

Reply:
ok those seem like good reasons for keeping it as is. 👍